src/Confluent.Kafka/Consumer.cs

```diff
@@ -312,13 +312,13 @@ public void Flush()
         public string MemberId
             => consumer.MemberId;
 
-        /// <summary>
+        /// <returns>
```

Comment:
not sure how auto-generated docs look but i suspect having something in the `summary` section is going to be good for that.